src/app/pages/courses-page/courses-page.component.html

```diff
@@ -1 +1,7 @@
-<p>courses-page works!</p>
+<app-breadcrumbs></app-breadcrumbs>
+
+<section class="courses-page-container">
+  <app-actions-section></app-actions-section>
+  <app-course-item *ngFor="let course of courses" [course]="course" (courseDelete)="onCourseDelete($event)"></app-course-item>
+  <app-load-more-panel></app-load-more-panel>
```

Comment:
this must accept an output handler: `onLoadMore()`

Reply:
Okay